src/ontodia/data/rdf/rdfCacheableStore.ts

```diff
@@ -0,0 +1,261 @@
+import {RDFStore, RDFGraph, createStore, createGraph, Node, Literal, NamedNode, Triple} from 'rdf-ext';
+import {Dictionary} from '../model';
+import {RDFCompositeParser} from './rdfCompositeParser';
+
+import {uniqueId} from 'lodash';
+
+const DEFAULT_STOREG_TYPE = 'text/turtle';
+const DEFAULT_STOREG_URI = 'ontodiaLocalData';
```

Comment:
`ontodiaLocalData` is not a valid IRI contrary to the constant name `DEFAULT_STOREG_URI` (it's missing a schema part).